dsl/src/commonMain/kotlin/it/unibo/collektive/messages/SentMessage.kt

```diff
@@ -0,0 +1,24 @@
+package it.unibo.collektive.messages
+
+import it.unibo.collektive.ID
+import it.unibo.collektive.stack.Path
+
+/**
+ * Types of messages sent by a device.
+ */
+sealed interface SentMessage
+
+/**
+ * Message sent from a device to other devices in the network.
+ * @param senderID: id of the device that sends the message.
+ * @param message: map <Path, value> of the message sent.
+ */
+data class IsotropicMessage(val senderID: ID, val message: Map<Path, *>) : SentMessage
```

Comment:
In other places like `ReceiveMessage`, you use `senderId` instead of `senderID`.
Make a choice and uniform the names.
To be strict with the camel case convention I prefer `senderId`, but it is up to you :)

Reply:
Understood! will update all the code with "Id" format.